platform/android/src/test/main.jni.cpp

```diff
@@ -48,54 +42,35 @@ struct Main {
 
 } // namespace
 
-// JNI Bindings to stub the android.util.Log implementation
-
-static jboolean isLoggable(JNIEnv* env, jni::jobject* clazz, jni::jstring* tag, jint level) {
-    return true;
-}
-
-static jint println_native(JNIEnv* env, jni::jobject* clazz, jint bufID, jint priority, jni::jstring* jtag, jni::jstring* jmessage) {
-    if (jtag == nullptr || jmessage == nullptr) {
-        return false;
-    }
-
-    std::string tag = jni::Make<std::string>(*env, jni::String(jtag));
-    std::string message = jni::Make<std::string>(*env, jni::String(jmessage));
-
-    return __android_log_print(priority, tag.c_str(), "%s", message.c_str());
-}
-
-static jint logger_entry_max_payload_native(JNIEnv* env, jni::jobject* clazz) {
-    return static_cast<jint>(4068);
-}
-
+// We're declaring the function, which is libandroid_runtime.so.
+// It is defined in AndroidRuntime.cpp:
+// https://github.com/android/platform_frameworks_base/blob/master/core/jni/AndroidRuntime.cpp
+// Once this symbol goes away, we'll have to revisit.
```

Comment:
This file is not part of the SDK; it is only used to bootstrap the core unit tests (with Google Test). If this symbol went away, the only thing that would break is launching the unit tests, but it does not impact in any way the function of a deployed version of Mapbox GL.

I've checked the past couple of Android versions and this symbol remained stable from (at least) 4.4 to 7.0.